Dockerfile.openssl

```diff
@@ -1,101 +1,87 @@
 FROM clearlinux:base as builder
 
-ARG QAT_DRIVER_RELEASE="qat1.7.l.4.10.0-00014"
-ARG QAT_ENGINE_VERSION="v0.5.46"
-ARG IPSEC_MB_VERSION="v0.54"
-ARG IPP_CRYPTO_VERSION="ippcp_2020u2"
+ARG QAT_DRIVER_RELEASE="qat1.7.l.4.11.0-00001"
+ARG QAT_ENGINE_VERSION="v0.6.1"
+ARG IPSEC_MB_VERSION="v0.55"
+ARG IPP_CRYPTO_VERSION="ippcp_2020u3"
 
 RUN swupd update && \
-    swupd bundle-add \
-    --skip-diskspace-check \
-    --no-boot-update \
-    devpkg-systemd \
-    devpkg-openssl \
-    c-basic \
-    curl \
-    git \
-    llvm9 \
-    python3-basic \
-    os-core-dev && \
-    git clone -b $QAT_ENGINE_VERSION https://github.com/intel/QAT_Engine && \
-    git clone -b $IPP_CRYPTO_VERSION https://github.com/intel/ipp-crypto && \
-    git clone -b $IPSEC_MB_VERSION https://github.com/intel/intel-ipsec-mb && \
-    mkdir QAT_Lib && cd QAT_Lib && \
-    curl -L https://01.org/sites/default/files/downloads/$QAT_DRIVER_RELEASE.tar.gz | tar zx
-
-WORKDIR /QAT_Lib
-
-RUN sed -i -e 's/cmn_ko$//' -e 's/lac_kernel$//' quickassist/Makefile && \
+    swupd bundle-add --skip-diskspace-check --no-boot-update \
+        os-core-dev \
+        c-basic \
+        llvm10 \
+        curl \
+        git \
+        python3-basic \
+        devpkg-systemd \
+        devpkg-openssl && \
+    curl -fsSL --output /usr/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v1.6.1/bazelisk-linux-amd64 && \
```

Comment:
looks like bazelisk version could be easily added via an ARG setting

Reply:
@mythi 

OK, done.